qemu/tests/blockdev_snapshot_chains.py

```diff
@@ -26,6 +26,7 @@ def prepare_snapshot_file(self):
             self.params["image_name_%s" % snapshot_tag] = snapshot_tag
             snapshot_format = params.get("snapshot_format", "qcow2")
             params["image_format_%s" % snapshot_tag] = snapshot_format
+            self.params["image_format_%s" % snapshot_tag] = snapshot_format
             image = sp_admin.volume_define_by_params(snapshot_tag, params)
             image.hotplug(self.main_vm)
 
```

Comment:
In 'verify_snapshot()', we need to add following lines 59/60 to let the following rebase work:

 55     def verify_snapshot(self):
 56         if self.main_vm.is_alive():
 57             self.main_vm.destroy()
 58         base_tag = self.base_tag
 59        ** image_format = self.params.get("image_format", "qcow2")
 60        ** self.params["image_format_%s" % base_tag] = image_format